pygeofilter/backends/cql2_json/evaluate.py

```diff
@@ -75,7 +75,7 @@ def like(self, node, *subargs):
 
     @handle(ast.IsNull)
     def isnull(self, node, arg):
-        return {"op": "isNull", "args": arg}
+        return {"op": "isNull", "args": [arg]}
```

Comment:
Should this be `{"op": op, "args": [*args]}` to handle multiple args?

Reply:
I don't believe `arg` would be a list here - from my understanding, the `isNull` predicate only accepts one expression; accordingly, the parser handling of `isNull` retrieves `args[0]` if `args` is a list. Happy to be corrected if I'm mistaken though!